mru/mru.go

```diff
@@ -1,7 +1,11 @@
-// This package implements a synchronized MRU (most recently used) map. Its a map that will hold at most cap key/value pairs
-// and will evict entries if: 1) adding an element would exceed cap, in this case the oldest existing element is removed to
-// make space for a new one. 2) an element's age exceeds maxage, the check is done when adding elements.
-// Age is defined as the duration between now and the time the element was last accessed (added or looked up).
+// This package implements a synchronized, MRU (most recently used) map. Its a map that holds at most cap key/value pairs
+// Entries are evicted entries if:
```

Comment:
📓 "entries" is repeated by mistake. Should be "Entries are evicted if:"

Reply:
fixed

Reply:
fixed